src/openrct2/rct2/S6Importer.cpp

```diff
@@ -1528,6 +1528,20 @@ std::unique_ptr<IParkImporter> ParkImporter::CreateS6(IObjectRepository& objectR
     return std::make_unique<S6Importer>(objectRepository);
 }
 
+static void show_error(uint8_t errorType, const rct_string_id& errorStringId)
+{
+    rct_string_id titleText = STR_UNABLE_TO_LOAD_FILE;
+    rct_string_id bodyText = errorStringId;
+    if (errorType == ERROR_TYPE_GENERIC)
+    {
+        titleText = errorStringId;
+        bodyText = 0xFFFF;
+    }
+    errorType = ERROR_TYPE_NONE;
```

Comment:
This assignment seems redundant.

Reply:
It was left over from when I copy-pasted the common parts out, I've taken it out in my latest commit